M2/Macaulay2/tests/normal/permutations.m2

```diff
@@ -1,5 +0,0 @@
-f = id_(ZZ^5)
-v = {1,2,4,3,0}
-w = {3,1,2,4,0}
-assert(f_(v_w)==(f_v)_w)
-assert(f_(v_w)==(f_v)*(f_w))
```

Comment:
Why is this test file being removed?

Reply:
It's moved to the package tests.

Reply:
Originally, I had tried to implement all of this directly into M2, i.e., not as a package, so I had written all of its tests here. Upon recommendation, I then tried to refactor all of permutations code I had written into a package and I simply forgot that M2 actually had this file to begin with, so it got deleted during the refactor.

Reply:
Oh, ok -- I see it in the package tests now.  But it doesn't use anything from the package, so should it stay where it was?

Reply:
I think it staying where it was makes sense as it uses M2's base functionality.